modules/nw-dpu-configuring-operator.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * networking/networking_operators/configuring-dpu-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-dpu-configuring-operator_{context}"]
+= Configuring the DPU Operator 
+
+Create a DpuOperatorConfig custom resource (CR) to deploy the components necessary to manage the host or the DPU-side cluster. The DPU Operator is deployed on the host-side cluster, and the DPU Operator manages the MicroShift clusters on each DPU:
```

Comment:
Create a DpuOperatorConfig custom resource (CR) on both the host-side cluster and on each of the dpu-side clusters. The DPU operator in each cluster will be activated after this CR is created. 

(Maybe this makes more sense since we don't cloud the clarity by mentioning microshift).

Reply:
Since we're in the "configuring the operator" section, we should move the part that explains that it's deployed on the host and dpu to the installation section.